raster-foundry/raster-foundry
# Add migration to delete duplicate unused landsat scenes

## Overview
Delete duplicate landsat scenes that are not being used by projects

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~

## Notes
There will probably still be a few duplicate landsat scenes left over - on production, at least 66 scenes with duplicates are being used in projects, and I don't touch them.

## Testing Instructions

 * On staging or production, run the inner select used to determine which scenes to delete and make sure the numbers make sense.

Closes #2526 
Closes https://github.com/azavea/raster-foundry-platform/issues/150
